### PR TITLE
Add "Control statements" section

### PR DESCRIPTION
**Pull request by ndresx**
This introduces a "Control statements" section to show how to format really long statements like `if` and `while` and what is the preferred way to split them up across multiple lines. Resolves https://github.com/airbnb/javascript/issues/1380.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1963,10 +1964,68 @@ Other Style Guides
...
+    }
+    ```
+
+
+**[⬆ back to top](#table-of-contents)**
+
+
 ## Comments
 
-  <a name="comments--multiline"></a><a name="17.1"></a>
```
Comment (ljharb) [inline README.md]: numbered links are there for legacy reasons and never change for any reason; please revert this (same throughout)

File: README.md [hunk 2]
```diff
@@ -1963,10 +1964,68 @@ Other Style Guides
...
-  - [17.1](#comments--multiline) Use `/** ... */` for multi-line comments.
+  <a name="comments--multiline"></a><a name="18.1"></a>
```
Comment (ljharb) [inline README.md]: revert this also

File: README.md [hunk 3]
```diff
@@ -1963,10 +1964,68 @@ Other Style Guides
...
+  - [18.1](#comments--multiline) Use `/** ... */` for multi-line comments.
```
Comment (ljharb) [inline README.md]: this can stay

**Comment by ndresx** [comment]
@ljharb Oh ok, should be good now. Not sure about this one though: `[21.1](#20.1) **Yup.** eslint:`?

File: README.md [hunk 4]
```diff
@@ -2569,7 +2628,7 @@ Other Style Guides
 ## Semicolons
 
   <a name="semicolons--required"></a><a name="20.1"></a>
-  - [20.1](#20.1) **Yup.** eslint: [`semi`](http://eslint.org/docs/rules/semi.html) jscs: [`requireSemicolons`](http://jscs.info/rule/requireSemicolons)
+  - [21.1](#20.1) **Yup.** eslint: [`semi`](http://eslint.org/docs/rules/semi.html) jscs: [`requireSemicolons`](http://jscs.info/rule/requireSemicolons)
```
Comment (ndresx) [inline README.md]: Thanks, yes! Have rebased it with master.

**Pull request merged**